### ISSUE TITLE
update failed for AnAction with ID=CppSupport.CompileSomeThing: com/intellij/openapi/actionSystem/DataKeys

### ISSUE THREAD

**Issue by mikz** [nicity/CppTools#6]
Hi, I get plenty of errors like this in console in basically every project I use. 

```
update failed for AnAction with ID=CppSupport.CompileSomeThing: com/intellij/openapi/actionSystem/DataKeys
java.lang.NoClassDefFoundError: com/intellij/openapi/actionSystem/DataKeys
	at com.advancedtools.cpp.actions.CompileCppAction.update(CompileCppAction.java:305)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:113)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:205)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:153)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.g(ActionMenu.java:281)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$400(ActionMenu.java:41)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:235)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1028)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1107)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:333)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:215)
	at javax.swing.JMenu.setSelected(JMenu.java:294)
	at com.apple.laf.ScreenMenu$1.run(ScreenMenu.java:117)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:199)
	at apple.awt.CToolkit$CPeerEvent.dispatch(CToolkit.java:1221)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
```

I'd like to provide more information, but no idea how.

**Comment by nicity** [comment]
What is your IntelliJ Idea / CppTools plugin versions?

**Comment by mikz** [comment]
RubyMine build 139.12000 with latest CppTools (0.8 build 7).

Also I should note that I do not have any c/cpp files in the projects an use it just because of Makefile highlighting.

**Comment by 3ddi** [comment]
Happens to me too, on windows. I am using the plugin only to parse the h/c files, but it occasionally throws exception and cfserver takes a lot of the cpu.

**Comment by nicity** [comment]
Fixed in build 0.8.8
